cmd/githubgen/main.go

```diff
@@ -206,3 +245,44 @@ LOOP:
 
 	return nil
 }
+
+func inList(list []string, id string) bool {
+	for _, m := range list {
+		if id == m {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Could it be possible to use maps instead?

Reply:
If we must? Do we care about performance all that much for this code?

Reply:
done